Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,67 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_you:
+    if not renpy.seen_label("mas_compliment_you_2"):
+        call mas_compliment_you_2
+    else:
+        call mas_compliment_you_3
+    return
+
+label mas_compliment_you_2:
+    m 1wubso "..."
+    m 1wubfb "[player]..."
+    
+    if mas_isMoniUpset(higher=True):
+        m 2hubfb "You have no idea what that means to me."
+        
+    elif mas_isMoniDis(True):    
+        m 1ekbfa "..."
+        m "Thank you."
+        
+    elif mas_isMoniBroken(True):
+        m 1rud "Hmmm... Thank you."
+        m 2eua "..."
```

Comment:
I think the expression here is a bit strange, considering this is in the broken path. You have what's probably supposed to be a sad expression in the previous line, and then this common neutral/happy expression with some silence.

Reply:
I wanted a semi-neutral expression, but I simply couldn't find one. It was meant as a "Broken Moni is slightly touched", but I couldn't find a good expression that would fit it.

Reply:
Maybe rksdla would better suit your needs.

Reply:
For the first or the second?

I added this one into the second spot, but I want to be sure before I finalize the commit.

Reply:
rksdld for the first one since I put the comment on that line. 

rksdla for this one since I put that comment on this line.

Reply:
Btw, which expression is rksdld? I cannot find it in the exp checker.